src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Compilation.cs

```diff
@@ -692,7 +692,19 @@ public IEnumerable<TypeDesc> ConstructedEETypes
             }
         }
 
-        public DependencyAnalyzerBase<NodeFactory> Graph => _graph;
+        public IEnumerable<TypeDesc> AllEETypes
+        {
+            get
+            {
+                foreach (var node in MarkedNodes)
+                {
+                    if (node is IEETypeNode)
```

Comment:
Note this will have duplicates if constructed and unconstructed forms exist. Probably fine.

Reply:
The infra must handle duplicates for other reasons already, so this is not a problem.